integr8ly/installation
# INTLY-307 install apicurito from template rebase

## What
Rebased version of #272 to include patches on master

## Verification Steps
* Run the installer
* Ensure Apicurito is installed instead of Apicurio
* Ensure Apicurito seems to work
* Run the webapp and ensure the ServiceInstance created links to Apicurito

## Checklist:
